### PR TITLE
Complaint model and serializer added

### PR DESCRIPTION
**Pull request by neeraj3029**
@samagragupta 
I have removed the get and post functions I created in the last Pull Request. I have added Complaint model, view and serialiser. Check comments

### CODE REVIEW

File: api/students/views.py [hunk 1]
```diff
@@ -1,45 +1,19 @@
 from django.shortcuts import render
 from rest_framework import viewsets, generics
-from .models import User
-from .serializers import UserSerializer
+from .models import User, Complaint
+from .serializers import UserSerializer, ComplaintSerializer
 
 # Create your views here.
-
 class UserView(viewsets.ModelViewSet):
```
Comment (neeraj3029) [inline api/students/views.py:7]: This View also provides the endpoint for accessing and deleting individual users, so there was no need for the `UserDetailView`, so I removed it

**Review by samagragupta — COMMENTED** [review]
We can also add the complaint status here.

**Comment by neeraj3029** [comment]
There is a status attribute in the complaint model

**Review by samagragupta — COMMENTED** [review]
LGTM
@rossoskull @akshi-11

**Pull request merged**